zmdocument.cls

```diff
@@ -430,39 +447,68 @@ overlay={\node at (frame.north west) {\createBoxIcon{#5}{#6}};}
    \setcounter{@authorsNumber}{\the@listSize}
    \setcounter{@columnNumber}{\the\numexpr(1 + \the@authorsNumber / \the@authorsByColumn)\relax}
    \thispagestyle{empty}
-   \begin{flushright}
-      \Large \bfseries \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\color{titlePageAuthorColor}\bsc{##1}}\\}
-      \ifnum\the@authorsNumber < \the\numexpr(1+\the@authorsByColumn)
+   \Large \bfseries \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\color{titlePageAuthorColor}\bsc{##1}}\\}
+   \ifnum\the@authorsNumber < \the\numexpr(1+\the@authorsByColumn)
+      \expandafter\docsvlist\expandafter{\@author}
+   \else
+      \begin{multicols}{\the@columnNumber}
          \expandafter\docsvlist\expandafter{\@author}
-      \else
-         \begin{multicols}{\the@columnNumber}
-            \expandafter\docsvlist\expandafter{\@author}
-         \end{multicols}
-      \fi
-   \end{flushright}
+      \end{multicols}
+   \fi
 }
 
 \renewcommand{\maketitle}{%
    \ifcsdef{@authorlink}{}{\authorlink{https://zestedesavoir.com/membres/voir}}
    \ifcsdef{@website}{}{\website{https://zestedesavoir.com}}
    \ifcsdef{@editor}{}{\editor{https://zestedesavoir.com}}
-   \ifcsdef{@editorlogo}{}{\editorlogo{zestedesavoir.png}}
+   \ifcsdef{@editorlogo}{}{\editorlogo{\@logoPath/full.svg}}
    \ifcsdef{@logo}{}{\logo{default_logo.png}}
+   \ifcsdef{@tutoLink}{}{\tutoLink{\@website}}
    \pagecolor{titlePageBackgroundColor}
    \color{titlePageTextColor}
    \clearpage
-   \formatAuthors
+   \newgeometry{top=1cm, bottom=1cm, right=1.5cm,left=1.5cm}
+
+   \begin{table}[h!]
    \begin{center}
-      \vspace*{\fill}
-      \includegraphics[max width=\textwidth, max height=4cm, keepaspectratio]{\@logo}\\
-      \rule{\linewidth}{0.2 mm} \\[0.4 cm]
+      \begin{tabular}{ p{0.7\textwidth} p{0.3\textwidth}  }
+         \vspace{0pt}
+         \begin{minipage}{0.7\textwidth}
+         \formatAuthors
+         \end{minipage}
+            &
+         \vspace{0pt}
+         \begin{minipage}[t]{0.3\textwidth}
+            \hspace{\fill}\@licence
+         \end{minipage}
+      \end{tabular}
+      \end{center}
+   \end{table}
+
+   \begin{center}
+      \vspace{4cm}
+      \externalCleanLink{\includegraphics[max width=\textwidth, max height=4cm, keepaspectratio]{\@logo}}{\@tutoLink}\\
       \huge\textbf{\@title}\\
-      \rule{\linewidth}{0.2 mm} \\[1 cm]
-      \large\textbf{Ce tutoriel est sous licence \@licence}\\
-      \vspace{\fill}
-      \includegraphics[max width=8cm, max height=2cm, keepaspectratio]{\@editorlogo}\\
-      \href{\@website}{\color{titlePageEditorColor}\@editor}
+      \color{chapterColor}\rule{\textwidth}{1 mm} \\
+      \ifthenelse{\isempty{\@date}}
+      {}
+      {\LARGE {\color{coverDateColor} \textbf{\@date}}}
+      \normalcolor
+      \vspace*{\fill}
+   \end{center}
+   \vspace*{2.1cm}
+   \begin{center}
+      \externalCleanLink{%
+\begin{minipage}{0.5\textwidth}
```

Comment:
Are you obliged to use minipage here ? (real  question ^^)

Reply:
If I didn't use Minipage env, it is because it will suck display

Reply:
I need a translation of what you mean by "suck display" (*aspirer l'affichage* ? *bousiller l'affichage* ?)

Reply:
Foiré l'affichage

Reply:
That is weird in this case, but if you say so, ok :)

Reply:
(ps: "break the display" or "break the alignment", or if you want to be less polite "fucked up the display"  :p "suck" is, in my opinion, only used like this in the expression "that sucks")